get-started/index.md

```diff
@@ -111,21 +111,21 @@ Consider this diagram comparing virtual machines to containers:
 
 ![Virtual machine stack example](https://www.docker.com/sites/default/files/VM%402x.png)
 
-Virtual machines run guest operating systems -- note the OS layer in each box.
-This is resource intensive, and the resulting disk image and application state is
-an entanglement of OS settings, system-installed dependencies, OS security
-patches, and other easy-to-lose, hard-to-replicate ephemera.
+Virtual machines run guest operating systems &#8212; note the OS layer in each
+box. This is resource intensive, and the resulting disk image and application
+state is an entanglement of OS settings, system-installed dependencies, OS
+security patches, and other easy-to-lose, hard-to-replicate ephemera.
 
 ### Container diagram
 
 ![Container stack example](https://www.docker.com/sites/default/files/Container%402x.png)
 
 Containers can share a single kernel, and the only information that needs to be
-in a container image is the executable and its package dependencies, which
-never need to be installed on the host system. These processes run like native
+in a container image is the executable and its package dependencies, which never
+need to be installed on the host system. These processes run like native
 processes, and you can manage them individually by running commands like `docker
-ps` -- just like you would run `ps` on Linux to see active processes. Finally,
-because they contain all their dependencies, there is no configuration
+ps` &#8212; just like you would run `ps` on Linux to see active processes.
+Finally, because they contain all their dependencies, there is no configuration
 entanglement; a containerized app "runs anywhere."
```

Comment:
Yeah I was just thinking that you can't run a Linux container on Windows Server using the Windows container kernel. I am sure you can just skip this feedback, I was just htinking "runs everywhere within reason". Also you can't run an armhf container on x86 architecture, etc. It's in the weeds.